### DISCUSSION TITLE
Wanna contribute to front end of pockethost.

### DISCUSSION THREAD

**Discussion by ammarrik** [pockethost/pockethost#176] [Q&A]
Hello @benallfree,

sorry for tagging your name, but I wanna contribute to the UI of pockethost. I also wanna contribute to business logic but want to start with UI. I'm new to contributions in open source repo (public repo) that's why I started the discussion if I'm permitted to contribute or not.

Is there something I can help to improve the UI or make it mobile responsive?

Thanks,
Ammar

**Comment by benallfree** [comment]
Hello, that is great news! Please check out #173 for a great way to get started as a contributor. 

If that doesn't seem like the right fit, please come back here and we can continue discussing.

**Reply by ammarrik** [reply]
I linked a pull request #177 with a fix.
